src/drivers/spektrum_rc/spektrum_rc.cpp

```diff
@@ -50,8 +50,11 @@
 #include <uORB/topics/input_rc.h>
 
 // Snapdraogon: use J12 (next to J13, power module side)
+#if defined(__PX4_QURT)
+#define SPEKTRUM_UART_DEVICE_PATH "/dev/tty-1"
+#else
 #define SPEKTRUM_UART_DEVICE_PATH "/dev/tty-3"
```

Comment:
It looks like it was passed on the command line and that was removed.

It is not a common element. Not all boards use it.

I would move it to the boards that  use it - as RC_SERIAL_PORT, if the scanning is done, But if it is if it is only for spektrum then move it as SPEKTRUM_UART_DEVICE_PATH to the boards that have to supply it.